Tibia/CharacterRepository.cs

```diff
@@ -12,25 +12,59 @@ public CharacterRepository()
         {
             equipmentRepository = new EquipmentRepository();
         }
-        private EquipmentRepository equipmentRepository = new EquipmentRepository();
-        public Character Retrieve(string characterName)
+        private EquipmentRepository equipmentRepository { get; set; }
+        public Character Get(string charName)
         {
-            var character = new Character(characterName);
-            character.CharacterName = characterName;
-            character.Equipment = equipmentRepository.Retrieve(characterName);
-            if(characterName == "Jacula")
+            var character = new Character();
+            character = CharBase.First(charac => charac.CharacterName == charName);
+            return character;
```

Comment:
1. Co jak nie ma chara o danym nicku
2. Nie potrzebujesz tworzyć nowew Character() bo i tak go nadpisujesz, mozesz zrobić to wszystko w jednej linii